internal/cli/atlas/backup/exports/jobs/decribe.go

```diff
@@ -79,13 +79,16 @@ func DescribeBuilder() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&opts.clusterName, flag.ClusterName, "", usage.ClusterName)
-	cmd.Flags().StringVar(&opts.bucketID, flag.BucketID, "", usage.BucketID)
+	cmd.Flags().StringVar(&opts.jobID, flag.JobID, "", usage.JobID)
+	cmd.Flags().StringVar(&opts.jobID, flag.BucketID, "", usage.JobID)
 
 	_ = cmd.MarkFlagRequired(flag.ClusterName)
-	_ = cmd.MarkFlagRequired(flag.BucketID)
+	_ = cmd.MarkFlagRequired(flag.JobID)
```

Comment:
I have marked this as required based on [this comment](https://github.com/mongodb/mongodb-atlas-cli/pull/1558#discussion_r1046864430), but thinking there's a possibility that could break backwards compatibility, will remove this change if so.

Tagging @gssbzn @andreaangiolillo for your thoughts

Reply:
Removed the requirement after local testing as it breaks backwards compatibility on existing requests that will be using the `--bucketId` flag